pages/index.js

```diff
@@ -1,7 +1,105 @@
+import { exercises } from "@/lib/exercises";
+import Image from "next/image";
+import styled from "styled-components";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <Header>
+        <h1>GymLog</h1>
+        <h2>Your ultimate Fitness platform</h2>
```

Comment:
It’s common practice to capitalize the main words in headlines or titles:

```suggestion
        <h2>Your Ultimate Fitness Platform</h2>
```